_tests/integration/version_test.go

```diff
@@ -22,7 +22,7 @@ format version: 12
 os: %s
 go: %s
 build number: 
-commit:`, version.VERSION, expectedOSVersion, runtime.Version())
+commit: (devel)`, version.VERSION, expectedOSVersion, runtime.Version())
```

Comment:
Where is this `(devel)` coming from?

Reply:
This comes from the Go runtime. For the integration tests we build the binary manually here:
https://github.com/bitrise-io/bitrise/blob/92bff8c14aac9b9e0cdba4c463bace3d9188521f/bitrise.yml#L35-L47

We just simply use `go build` to do so and in that case Go uses `devel`. My thinking was that we never use this in our regular flow so it is fine to leave it here.